src/main/java/be/syntra/devshop/DevshopBack/controllers/ProductController.java

```diff
@@ -4,43 +4,52 @@
 import be.syntra.devshop.DevshopBack.models.CategoryList;
 import be.syntra.devshop.DevshopBack.models.ProductDto;
 import be.syntra.devshop.DevshopBack.models.ProductList;
+import be.syntra.devshop.DevshopBack.models.SearchModelDto;
 import be.syntra.devshop.DevshopBack.services.CategoryService;
 import be.syntra.devshop.DevshopBack.services.ProductService;
+import be.syntra.devshop.DevshopBack.services.SearchService;
+import be.syntra.devshop.DevshopBack.services.utilities.ProductMapperUtility;
+import be.syntra.devshop.DevshopBack.services.utilities.SearchModelMapperUtility;
+import lombok.extern.slf4j.Slf4j;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
+import java.util.List;
 
+@Slf4j
 @RestController
 @RequestMapping("/products")
 public class ProductController {
 
     private final ProductService productService;
     private final CategoryService categoryService;
+    private final SearchService searchService;
+    private final SearchModelMapperUtility searchModelMapperUtility;
```

Comment:
One of these days one of you is going to have to rename these so they aren't called "Utility" :)
Same goes for front-end mappers

Reply:
will do that now